strings/british_talk.json

```diff
@@ -0,0 +1,118 @@
+{
+
+    "words": {
+        "id": "BBM Pin",
+        "fuck": "fock",
+        "shit": "shite",
+        "dick": "johny",
+        "arrested": "nicked",
+        "arrest": "nick",
+        "security":"coppers",
+        "shitsec":"coppers",
+        "shitcurity":"coppers",
+        "captain": "queen",
+        "cap": "queen",
+        "kill": ["shank", "bang"],
+        "killed": ["shanked", "banged"],
+        "murder": ["shank", "bang"],
+        "murdered": ["shanked", "banged"],
+        "stab": ["shank", "bang"],
+        "stabbed": ["shanked", "banged"],
+		"robust": ["shank", "bang"],
+        "robusted": ["shanked", "banged"],
```

Comment:
```suggestion
        "robusted": ["shanked", "banged", "clobbered"],
```

Reply:
I've lived in England most of my life and I've not heard bang/banged before except when referring to someone being shot